test/client/scraper/test_drive_metadata.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import json
+from client.query_utils import QueryParams
+from client.constants import ResponseType
+import client.api_handler
+import client.api
+
+
+class TestDriveMetadata(unittest.TestCase):
+
+    test_url = "https://docs.google.com/document/d/1tPzlb1PmLgRWwePwxH4OXsp5UrKHa4BZMl0ho4sM8Sk/edit?usp=sharing"
```

Comment:
Yeah I would just make a new document with filler text and put it in the saavy google drive account rather than any of our personal ones